lib/util/patterns/letters.js

```diff
@@ -0,0 +1,10 @@
+/**
+ * @fileoverview Pattern for detecting any letter (even letters outside of
+ * ASCII).
+ * @author Kevin Partington
+ */
```

Comment:
Thanks, should I change the author information to match who generated it for JSCS? (And what about licensing?)

Or do you want me to regenerate this myself?

Sorry, the author field is misleading right now.